src/capi-single-paid/index.js

```diff
@@ -65,4 +67,38 @@ function checkIcon(responseJson) {
     responseJson.videoTag ?
         videoIcon :
         '';
-};
+}
+
+function addSourceset(responseJson) {
```

Comment:
Are these function shared with another component? If yes, might be useful to put them in sth like `_shared/js/capi`?